doc/intl/pt_PT/README.md

```diff
@@ -0,0 +1,97 @@
+
+**Website:** [dogecoin.com](https://dogecoin.com)
+
+## Utilização 💻
+
+Para começares a tua jornada com o Dogecoin Core, ve o [manual de instalação](INSTALL.md) e o [guia para iniciantes](doc/getting-started.md).
+
+A API JSON-RPC incluida no Dogecoin Core é auto documentada e pode ser vista com o comando `dogecoin-cli help`, mais informações detalhadas sobre cada comando podem ser encontradas utilizando `dogecoin-cli help <command>`. Alternativamente, veja a [documentação do Bitcoin Core](https://developer.bitcoin.org/reference/rpc/) - que implementa um protocolo similar - para conseguir uma versão navegável.
+
+### Quais as portas
+
+O Dogecoin Core utiliza por padrão a porta `22556` para comunicação em rede
+ponto-a-ponto que é necessária para sincronizar a blockchain da "rede principal",
+(mainnet), e se manter actualizadadas novas transações e blocos. Adicionalmente a
+porta JSONRPC pode ser aberta que por defeito é `22555` para a rede principal.
+É fortemente recomendado a não expor as portas RPC publicamente na internet.
+
+|  Função  | mainnet | testnet | regtest |
+| :------- | ------: | ------: | ------: |
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+## Desenvolvimento continuo - Plano, a Lua 🌒
+
+O Dogecoin Core é um programa de código aberto gerido pela comunidade. O processo de desenvolvimento é aberto e visivel publicamente; qualquer um pode ver, discutir e trabalhar no programa.
+
+Recursos principais de Desenvolvimento:
+
+* [Projetos do Github](https://github.com/dogecoin/dogecoin/projects) é utilizado para conduzir trabalhos planeado ou que estejam em desenvolvimento para as próximas atualizações.
+* [Discussão do Github](https://github.com/dogecoin/dogecoin/discussions) é usado para discutir sobre as funcionalidades, planeadas ou não, relacionadas ao desenvolvimento do programa Dogecoin Core, os protocolos adjacentes e o criptoativo DOGE.
+* [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
+
+### Estratégia das Versões
+Os numeros de compilação seguem a seguinte semantica:  ```major.minor.patch```
+
+### Ramificações
+Este repositório possui 3 principais ramificações (branchs), são estas:
+
+- **master:** Estável, contém a ultima versão da ultima atualização principal *major.minor*.
+- **maintenance:** Estável, contém a ultima versão de atualizações anteriores, que ainda estão em manutenção. Formato: ```<version>-maint```
+- **development:** Instável, contém código novo para atualizações planeadas. Formato: ```<version>-dev```
+
+*As ramificações Master e Maintenance, são exclusivamente mutáveis por lançamento. Atualizações*
+*planeadas terão sempre uma ramificação de desenvolvimento e as solicitações de inclusão deverão ser*
+*encaminhadas por meio destas. Ramificações de manutenção existem apenas para **correção de bugs,***
+*por favor, encaminhem novos recursos na ramificação de desenvolvimento com a versão mais alta.*
```

Comment:
not trying to be picky with the `atualização` but just wanted to throw a light on it to keep consistency with line 32 `"actualizadas"`

```suggestion
- **master:** Estável, contém a ultima versão da ultima actualização principal *major.minor*.
- **maintenance:** Estável, contém a ultima versão de actualizações anteriores, que ainda estão em manutenção. Formato: ```<version>-maint```
- **development:** Instável, contém código novo para actualizações planeadas. Formato: ```<version>-dev```

*As ramificações Master e Maintenance, são exclusivamente mutáveis por lançamento. Actualizações*
*planeadas terão sempre uma ramificação de desenvolvimento e as solicitações de inclusão deverão ser*
*encaminhadas por meio destas. Ramificações de manutenção existem apenas para **correção de bugs,***
*por favor, encaminhem novos recursos na ramificação de desenvolvimento com a versão mais alta.*
```

Reply:
yep! even better actually, good job :)